RampMotor.java

```diff
@@ -0,0 +1,43 @@
+package org.firstinspires.ftc.teamcode.robot;
+
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.DcMotorController;
+import com.qualcomm.robotcore.hardware.DcMotorImpl;
+import com.qualcomm.robotcore.util.Range;
+
+public class RampMotor extends DcMotorImpl {
+
+    private static final double INCREMENT = 0.05, MIN = -0.8, MAX = 0.8;
```

Comment:
Why the max power is 0.8?

Reply:
Debugging purposes, the min and max will be set to 1.0